backend/fms_core/filters.py

```diff
@@ -0,0 +1,58 @@
+from django.db.models import Q
+
+from .models import Container, Sample, Individual
+
+import django_filters
+
+from .viewsets._constants import (
+    _container_filterset_fields,
+    _sample_filterset_fields,
+    _individual_filterset_fields,
+)
+
+class ContainerBatchFilter(django_filters.FilterSet):
+    barcode = django_filters.CharFilter(field_name="barcode", method="batch_barcode_filter")
+    name = django_filters.CharFilter(field_name="name", method="batch_name_filter")
+
+    class Meta:
+        model = Container
+        fields = _container_filterset_fields
```

Comment:
are these fields already filled by the call to filterset_fields in the viewset ?